src/main/scala/no/ndla/oembedproxy/OEmbedProxyProperties.scala

```diff
@@ -32,10 +32,20 @@ object OEmbedProxyProperties {
     "prod" -> "https://ndla.no/oembed"
   ).getOrElse(Environment, s"https://$Environment.ndla.no/oembed")
 
+  val ListingFrontendOembedServiceUrl = Map(
+    "local" -> "http://localhost:30020/oembed",
+    "prod" -> "https://liste.ndla.no/oembed"
+  ).getOrElse(Environment, s"https://liste.$Environment.ndla.no/oembed")
+
+  val ListingFrontendApprovedUrls = Map(
+    "local" -> List("http://localhost:30020/*"),
+    "prod" -> List("https?://liste.ndla.no/*")
+  ).getOrElse(Environment, List(s"https?://www.liste.$Environment.ndla.no/*", s"https?://liste.$Environment.ndla.no/*"))
```

Comment:
Fremdeles ikkje behov for www på denne linja heller.

Reply:
stemmer, gelmte denne